dockers/docker-dhcp-relay/dhcp-relay.programs.j2

```diff
@@ -1,17 +1,23 @@
 [group:isc-dhcp-relay]
 programs=
 {%- set add_preceding_comma = { 'flag': False } %}
-{% for vlan_name in VLAN_INTERFACE %}
 {# Append DHCPv4 agents #}
+{% if ipv4_num_relays.count > 0 %}
+{% for vlan_name in VLAN_INTERFACE %}
 {% if VLAN and vlan_name in VLAN and 'dhcp_servers' in VLAN[vlan_name] and VLAN[vlan_name]['dhcp_servers']|length > 0 %}
 {% if add_preceding_comma.flag %},{% endif %}
 {% set _dummy = add_preceding_comma.update({'flag': True}) %}
 isc-dhcpv4-relay-{{ vlan_name }}
 {%- endif %}
-{# Append DHCPv6 agents #}
+{% endfor %}
+{% endif %}
+{# Append DHCPv6 agent #}
+{% if ipv6_num_relays.count > 0 %}
+{% for vlan_name in VLAN_INTERFACE %}
 {% if VLAN and vlan_name in VLAN and 'dhcpv6_servers' in VLAN[vlan_name] and VLAN[vlan_name]['dhcpv6_servers']|length > 0 %}
-{% if add_preceding_comma.flag %},{% endif %}
-{% set _dummy = add_preceding_comma.update({'flag': True}) %}
-isc-dhcpv6-relay-{{ vlan_name }}
+{% set _dummy = v6_vlan_list.append( vlan_name ) %}
 {%- endif %}
 {% endfor %}
+{% if add_preceding_comma.flag %},{% endif %}
+isc-dhcpv6-relay-{{ v6_vlan_list|join("-") }}
+{% endif %}
```

Comment:
@shlomibitton have you missed a newline?

Reply:
no, this is intentionally this way, if I will add a new line it will add it in the output file also and this is not desired